Simonyi/Troubleshooting/MTCS/MTDome/MTDome-to-Home.rst

```diff
@@ -0,0 +1,93 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Paulina Venegas*
+
+.. _Cam-04: https://unifi.ui.com/consoles/D021F9521E8800000000063CF3E6000000000686EFB60000000061C81AC6:201406364/protect/dashboard/all
+
+.. _Troubleshooting-MTCS-MTDome-MTDome-re-Home:
+
+##############
+MTDome re-Home
+##############
+
+.. _MTDome-MTDome-re-Home-Overview:
+
+Overview
+========
+The dome needs re-homing if the reference azimuth position is lost, and the encoders are not reading the real position of the dome azimuth (as seen in the camera).
+
+.. note::
+  This is a temporary situation which will be resolved once EIE updates the azimuth rotation control software. A bar code will be installed by EIE which will provide absolute positioning for the ``MTDome``.
+
+  This process already has started but will take a few more months. 
+  
+  Once the upgrade is completed, this procedure can be deprecated.
+..
+
+
+.. _MTDome-MTDome-re-Home-Procedure-Error-Diagnosis:
+
+Error Diagnosis
+===============
+
+* While the dome is not positioned where it should be, the images might be vignetted by the dome. In more extreme cases, no images show at all if the cameras are completely covered by the dome.
+* The azimuth position reported by the dome is not the same as the one seen in the camera.
```

Comment:
Needs clarification -- 
1. where do you check the MTDome azimuth position? 
2. How do you check the MTDome azimuth position?

Needs a picture from the Unifi Camera to show how it actually looks through webcam.